app/screens/draft_scheduled_post_options/draft_scheduled_post_options.test.tsx

```diff
@@ -1,81 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Why are we removing this tests completely? The file it is related to doesn't seem to have been deleted nor updated.

Reply:
Will cover this test in a separate PR; that's the reason why it was removed. Created a JIRA ticket for it: https://mattermost.atlassian.net/browse/MM-63386